src/Sql/dbo/Tables/Project.sql

```diff
@@ -0,0 +1,15 @@
+﻿IF OBJECT_ID('[dbo].[Project]') IS NULL
+BEGIN
```

Comment:
The `dbo` object should not have these checks.

Reply:
Remove IF OBJECT_ID('[dbo].[Project]') IS NULL
BEGIN and GO at the end of the file as well

Reply:
@Hinton I have a PR with this fix here: https://github.com/bitwarden/server/pull/2215